src/function/mtk.jl

```diff
@@ -67,7 +67,33 @@ function instantiate_function(f, x, adtype::AutoModelingToolkit, p,
         H .= res * v
     end
 
-    cons = (res, θ) -> f.cons(res, θ, p)
+    expr = symbolify(ModelingToolkit.Symbolics.toexpr(ModelingToolkit.equations(sys)))
```

Comment:
Do we need these after https://github.com/SciML/Optimization.jl/pull/432?

Reply:
No